### ISSUE TITLE
TH1Merger::CheckForDuplicateLabels error in workflow 4.6

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#39737]
Workflow 4.6 step 3 fails in CMSSW_12_6_X_2022-10-14-1100 with
```
----- Begin Fatal Exception 14-Oct-2022 16:12:05 CEST-----------------------
An exception of category 'FatalRootError' occurred while
   [0] Calling InputSource::readRun_
   Additional Info:
      [a] Fatal Root Error: @SUB=TH1Merger::CheckForDuplicateLabels
Histogram GTErrorFlag has duplicate labels in the x axis. Bin contents will be merged in a single bin

----- End Fatal Exception -------------------------------------------------
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/el8_amd64_gcc10/CMSSW_12_6_X_2022-10-14-1100/pyRelValMatrixLogs/run/4.6_MinimumBias2010A+MinimumBias2010A+RECOSKIMALCA+HARVESTDR1/step3_MinimumBias2010A+MinimumBias2010A+RECOSKIMALCA+HARVESTDR1.log#/

**Comment by makortel** [comment]
assign dqm

**Comment by makortel** [comment]
The IB had two PRs merged in, https://github.com/cms-sw/cmssw/pull/39472 and https://github.com/cms-sw/cmssw/pull/39716. On a cursory look neither look like a potential culprit to me (no updates in externals), so I'm puzzled what exactly would be causing this error to appear in CMSSW_12_6_X_2022-10-14-1100

**Comment by perrotta** [comment]
urgent
(marking urgent the issues affecting relvals in the IBs)

**Comment by perrotta** [comment]
I don't remember how we finally managed to fix it, but the workflow is not failing any more in the IBs
This can probably get closed, isn't it?

**Comment by smuzaffar** [comment]
@perrotta , this was fix via https://github.com/cms-sw/cmssw/pull/39853 and can be closed now

**Comment by perrotta** [comment]
> @perrotta , this was fix via #39853 and can be closed now

Thank you @smuzaffar for adding the reference to the fix!
Let close, then.
